xds/src/test/java/io/grpc/xds/internal/security/trust/XdsTrustManagerFactoryTest.java

```diff
@@ -105,6 +107,25 @@ public void constructor_fromRootCert()
         .isEqualTo(CertificateUtils.toX509Certificates(TlsTesting.loadCert(CA_PEM_FILE))[0]);
   }
 
+  @Test
+  public void constructor_fromSpiffeMap()
+      throws CertificateException, IOException, CertStoreException {
+    X509Certificate x509Cert = TestUtils.loadX509Cert(CA_PEM_FILE);
+    CertificateValidationContext staticValidationContext = buildStaticValidationContext("san1",
+        "san2");
+    XdsTrustManagerFactory factory = new XdsTrustManagerFactory(ImmutableMap
```

Comment:
Can we have a second test where there is (a) more than 1 key in the map, and (b) more than one value for one of the keys?